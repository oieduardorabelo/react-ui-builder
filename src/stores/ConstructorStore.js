const storeState = {
  sections: [],
}

export default {
  getState() {
    return storeState
  },
}
