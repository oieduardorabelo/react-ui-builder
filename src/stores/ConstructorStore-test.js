import ConstructorStore from './ConstructorStore'

describe('ConstructorStore', () => {
  it('retursn Section 1 correctly', () => {
    const actual = ConstructorStore.getState()
    expect(actual).toMatchSnapshot()
  })
})
