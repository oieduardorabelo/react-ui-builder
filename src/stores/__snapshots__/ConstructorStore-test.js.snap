// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConstructorStore retursn Section 1 correctly 1`] = `
Object {
  "sections": Array [],
}
`;
